python/caffe/test/test_net.py

```diff
@@ -2,6 +2,7 @@
 import tempfile
 import os
 import numpy as np
+import six
```

Comment:
Please add six to the requirements.txt since it isn't always bundled as far as I know.